src/main/java/com/joss/conductor/mobile/ConductorConfig.java

```diff
@@ -443,4 +458,16 @@ private void putCustomCapabilities(Object keyCustomCapabilities) {
     private void putCustomCapabilities(Map<String, Object> customCapabilities) {
         this.customCapabilities.putAll(customCapabilities);
     }
+
+    public SauceOnDemandAuthentication getSauceAuthentication(String sauceUserName, String sauceAccessKey) {
+
+        if (sauceUserName == null) {
+            throw new InvalidArgumentException("Env var for SAUCE_USERNAME cannot be null");
```

Comment:
Update the error message to use the `config.yaml` and for they key `sauceUserName` and `sauceAccessKey` fields.